src/System.CommandLine.Tests/TrimmingTestApp/TrimmingTestApp.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<PublishTrimmed>true</PublishTrimmed>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<SystemCommandLineDllPath Condition="'$(SystemCommandLineDllPath)'==''">..\..\System.CommandLine\bin\Release\netstandard2.0\System.CommandLine.dll</SystemCommandLineDllPath>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<TrimmableAssembly Include="System.CommandLine" />
```

Comment:
I don't think this is needed since you set `IsTrimmable=true` in the assembly.

Reply:
Don't you want `TrimmerRootAssembly` instead?

[https://docs.microsoft.com/en-us/dotnet/core/deploying/trimming/prepare-libraries-for-trimming](https://docs.microsoft.com/en-us/dotnet/core/deploying/trimming/prepare-libraries-for-trimming)